.docker/scripts/init-ext.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+##
+# Install current extension.
+#
+set -e
+
+. /app/ckan/default/bin/activate
+
+pip install -r "/app/requirements.txt"
+pip install -r "/app/requirements-dev.txt"
+python setup.py develop
+
+# Validate that the extension was installed correctly.
+if ! pip list | grep ckanext-qgov > /dev/null; then echo "Unable to find the extension in the list"; exit 1; fi
```

Comment:
For another time, we should look at breaking out the plugin names so these scripts are even more generic.

Reply:
Done.